### PR TITLE
[QA-206] Addition of Multi-Select Test Suite

### PR DESCRIPTION
**Pull request by kiran-tripathy**
## Summary

Automated Scripts for Multi-Select Case List Tests

## Description

- Corresponding Test Plan: [Multi-Select Case List [Master]](https://docs.google.com/spreadsheets/d/1dCcjfufT4t0J_SPwRCkR0cEYfegg18hz6iPcgMOqpes/edit#gid=1232302962)
- In addition disabled success mails for case search and multiselect

### Link to Jira Ticket (if applicable)

[https://dimagi-dev.atlassian.net/browse/QA-206](https://dimagi-dev.atlassian.net/browse/QA-206 )

## QA Checklist

- [x] Label(s) and Assignee added
- [ ] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

File: Features/CaseSearch/test_cases/conftest.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 
 @pytest.fixture(scope="session")
-def environment_settings_casesearch():
+def environment_settings_multiselect():
```
Comment (kbo001) [inline Features/CaseSearch/test_cases/conftest.py:13]: I am a bit confused here. Why is this renamed as multiselect but the one under multiselect is named as case search?

File: Features/MultiSelect/test_cases/conftest.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+from common_utilities.fixtures import *
+
+""""This file provides fixture functions for driver initialization"""
+
+global driver
+
+
+@pytest.fixture(scope="session")
+def environment_settings_casesearch():
+    """Load settings from os.environ
```
Comment (kbo001) [inline Features/MultiSelect/test_cases/conftest.py:14]: Here it says casesearch but its under multiselect, a bit confusing

File: Features/MultiSelect/user_inputs/multiselect_user_inputs.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+""""Contains test data that are used as user inputs across various areass used in Case Serach"""
+
+
+class MultiSelectUserInput:
+    """User Test Data"""
+
+    """App Names"""
+    multiselect_app_name = "Multi-Select"
+
+    """Menu Names"""
```
Comment (kbo001) [inline Features/MultiSelect/user_inputs/multiselect_user_inputs.py:10]: Just some cosmetic changes. Name the variables starting with lower case.

**Review by kbo001 — CHANGES_REQUESTED** [review]

File: Features/MultiSelect/test_cases/conftest.py (hunk 2 above)
Comment (kiran-tripathy) [reply Features/MultiSelect/test_cases/conftest.py:14]: My intentions just hit the wrong file. 🗡️  Updated in https://github.com/dimagi/dimagi-qa/pull/209/commits/0634ed543e3e5846a8be5e8b7292a2e6a8f513b2

File: Features/CaseSearch/test_cases/conftest.py (hunk 1 above)
Comment (kiran-tripathy) [reply Features/CaseSearch/test_cases/conftest.py]: Updated in https://github.com/dimagi/dimagi-qa/commit/0634ed543e3e5846a8be5e8b7292a2e6a8f513b2

File: Features/MultiSelect/user_inputs/multiselect_user_inputs.py (hunk 3 above)
Comment (kiran-tripathy) [reply Features/MultiSelect/user_inputs/multiselect_user_inputs.py:10]: https://github.com/dimagi/dimagi-qa/pull/209/commits/46bed4f851a154e7e97c440d84d72572c1ef173e

**Review by kbo001 — APPROVED** [review]
